content/dstu2/patient.md

```diff
@@ -7,22 +7,33 @@ title: Patient | DSTU 2 API
 * TOC
 {:toc}
 
+## Terminology Bindings
+
+<%= terminology_table(:patient, :dstu2) %>
+
+## Extensions
+* [Time of day of birth]
+
 ## Search
 
 Search for Patients that meet supplied query parameters:
 
     GET /Patient?:parameters
 
+_Implementation Notes_
+
+* The [Patient.animal] and [Patient.link] Is-Modifier fields are not supported and will not be returned.
```

Comment:
You can refer to these as "modifier elements" rather than Is-Modifier fields.  When you hover over the ?! symbol next to these fields in the spec it indicates that the field is a modifier element, and [Grahame's blog](http://www.healthintersections.com.au/?p=2426) refers to them as modifier elements.  "is-modifier" is the boolean property that they set behind the scenes in the underlying spec excel spreadsheets, but they're just called modifier elements in reference.

This would mean updating the line for DocumentReference, too.